2020/cmd/08/vm.go

```diff
@@ -81,10 +85,12 @@ func parseInstruction(s string) (instr Instruction, err error) {
 }
 
 type Program struct {
+	// QUESTION: is `[]Instruction` ok, or should I do `[]*Instruction`?
```

Comment:
It is ok. Basically it depends on multiple factors which is better. I would use `[]Instruction` here, because having less pointers is simpler.